scylla-cql/src/frame/response/result.rs

```diff
@@ -52,9 +52,9 @@ pub struct TableSpec<'a> {
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum ColumnType<'frame> {
     Native(NativeType),
-    List(Box<ColumnType<'frame>>),
-    Map(Box<ColumnType<'frame>>, Box<ColumnType<'frame>>),
-    Set(Box<ColumnType<'frame>>),
+    Collection {
+        type_: CollectionType<'frame>,
+    },
```

Comment:
:pick: `type_` looks awful. Let's use `typ` instead, as in many other places in the driver.